<launch>

    <arg name="show_plots"             default="false"/>
    <arg name="track_time"             default="false"/>
    <arg name="output_file"            default="$(find logger)/output_data"/>
    <arg name="output_folder"          default=""/>

    <arg name="parent_frame"           default="odom"/>
    <arg name="robot_frame"            default="base_link"/>
    <arg name="kinetic_model_frame"    default="model_link"/>
 
    <node pkg="logger" type="logger_node.py" name="logger" output="screen">
        <param name="show_plots"           value="$(arg show_plots)" />
        <param name="track_time"           value="$(arg track_time)" />
        <param name="output_file"          value="$(arg output_file)" />
        <param name="output_folder"        value="$(arg output_folder)" />
        <param name="parent_frame"         value="$(arg parent_frame)" />
        <param name="robot_frame"          value="$(arg robot_frame)" />
        <param name="kinetic_model_frame"  value="$(arg kinetic_model_frame)" />
    </node>

 
</launch>